crmsh/bootstrap.py

```diff
@@ -1846,6 +1889,9 @@ def update_nodeid(nodeid, node=None):
         else:
             corosync.set_value("totem.nodeid", nodeid)
 
+    atexit.register(os.remove, COROSYNC_CONF_ORIG)
+    shutil.copy(corosync.conf(), COROSYNC_CONF_ORIG)
```

Comment:
Would it be better to use `tempfile.mktemp()` to create the temporary file, just in case this would accidentally override any existing files of users ? AFAICT, crmsh uses tempfile to create temporary things for other purposes as well.

Reply:
Changed. Use `tmpfiles.create` to create temporary file.
Thanks!

Reply:
Already changed for crmsh-4.1 and crmsh-3.0 branche PRs